### PR TITLE
feat: add Scrollbar css token

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | feat: add Scrollbar css token（已回滚） |
| 🇨🇳 Chinese | feat: 新增 Scrollbar 相关的 token（已回滚） |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/theme/themes/shared/genScrollbarToken.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { ScrollBarToken } from '../../interface/maps';
+
+function genScrollbarToken(): ScrollBarToken {
+  return {
+    scrollbarColor: 'auto',
+    scrollbarWidth: 'auto',
+    scrollbarGutter: 'auto',
```
Comment (afc163) [inline components/theme/themes/shared/genScrollbarToken.ts:7]: 在哪里使用这几个 token？

Comment (li-jia-nan) [reply components/theme/themes/shared/genScrollbarToken.ts:7]: https://github.com/ant-design/ant-design/pull/48088#discussion_r1539292192

Comment (afc163) [reply components/theme/themes/shared/genScrollbarToken.ts:7]: 我是说只看到定义，没看到具体哪个组件进行了消费。

Comment (li-jia-nan) [reply components/theme/themes/shared/genScrollbarToken.ts:7]: 这里没用到，在 master 分支需要用到，先做成 token，等 feature 合并进去，就把 master 分支相关的属性换成 token

**Review by afc163 — APPROVED** [review]

File: components/theme/interface/maps/scrollbar.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export interface ScrollBarToken {
+  /**
+   * @desc 滚动条颜色
+   * @descEN Scrollbar color
+   * @default auto
+   * @since 5.16.0
+   */
+  scrollbarColor: string;
```
Comment (MadCcc) [inline components/theme/interface/maps/scrollbar.ts:8]: 这个不符合 Design Token 的命名规范，让 @arvinxx 看看
这个命名方式比较适合放在 component token 里

Comment (li-jia-nan) [reply components/theme/interface/maps/scrollbar.ts:8]: 回滚了

Comment (li-jia-nan) [reply components/theme/interface/maps/scrollbar.ts:8]: - #48115

**Pull request merged**